### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by TomMichielsD2L**
Github actions instead of Travis

### CODE REVIEW

**Review by BraydonKains — APPROVED** [review]

**Review by julianpettit — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: CI
+on: pull_request
+
+jobs:
+  test:
+    name: Test
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Brightspace/third-party-actions@actions/checkout
```
Comment (josephsaffron) [inline .github/workflows/ci.yml:10]: should you be slotting the uses under the step that uses it?  Or does it not matter?

Comment (josephsaffron) [reply .github/workflows/ci.yml:10]: Or I guess this is just an unnamed step

Comment (TomMichielsD2L) [reply .github/workflows/ci.yml:10]: I will name these steps

**Pull request merged**